kili/cli/project/member/list_.py

```diff
@@ -0,0 +1,76 @@
+"""CLI's project list subcommand"""
+
+from typing import Dict, List, Optional, cast
+import click
+
+import pandas as pd
+from tabulate import tabulate
+
+from kili.client import Kili
+from kili.cli.common_args import Options
+
+ROLE_ORDER = {
+    v: i for i, v in enumerate(["ADMIN", "TEAM_MANAGER", "REVIEWER", "LABELER"])
+}
+
+
+def role_order(col: pd.Series) -> pd.Series:
+    """ordering pandas series by custom role order"""
+    out = col
+    # apply custom sorting only to column one:
+    if col.name == "ROLE":
+        out = col.map(ROLE_ORDER)
+    # default text sorting is about to be applied
+    return out
+
+
+@click.command()
+@Options.api_key
+@Options.endpoint
+@click.option('--project-id', type=str, required=True,
+              help='Id of the project to list members of')
+@Options.tablefmt
+def list_members(api_key: Optional[str],
+                 endpoint: Optional[str],
+                 project_id: str,
+                 tablefmt: str):
+    """
+    List the members of the project
+
+    \b
+    !!! Examples
+        ```
+        kili project member list --project-id <project_id> ----stdout-format pretty
+        ```
+
+    """
+    kili = Kili(api_key=api_key, api_endpoint=endpoint)
+    users = cast(
+        List[Dict], kili.project_users(
+            project_id=project_id,
+            fields=[
+                'id', 'role', 'activated',
+                'user.email', 'user.id',
+                'user.firstname', 'user.lastname',
+                'user.organization.name',
+            ],
+            disable_tqdm=True))
+    users = pd.DataFrame(users)
+    users.rename(columns={'id': 'ROLE_ID', 'role': 'ROLE',
```

Comment:
ROLE_ID is a bad name.
It should either be PROJECT_USER_ID or MEMBER_ID. 
In the front we use "member", in the API, we use 'Project User". Last time we said that we should rather use "member" to speak about a project user. Then MEMBER_ID ? @PierreLeveau do you agree ?

Reply:
I put this in the top comment : 
I kept the role_id (or project_user_id) as we put it in the specs, but actually, I am for removing it: even in the labeling authors, the user_id is used and not the project_user_id (the only use I can find is for the delete_user mutation). wdyt?
I vote to remove it.

Reply:
MEMBER_ID is too confusing with the real user_id I think.

Reply:
for sure ROLE_ID is a bad name because the id does not concern a role but a project_user.
in our architecture, we speak about a user and a project_user.
In the front, we speak about the user of an organization and the member of a project.
Why not keeping this here if it is what we expose to clients ?

Reply:
In fact, I agree with you Florian. Having both a user_id and a project_user_id can be confusing for a customer 
@PierreLeveau we could think to only keep one (rather the user_id) as function arguments in the SDK V3, and thus to only give the user_id here.

Reply:
Then I will remove it !